server.py

```diff
@@ -325,7 +325,8 @@ def run_addfields_data():
         typeVal = query_dict.pop('type')
 
     if typeVal == "bug":
-        sql = "update alerts set bug='%s',status='Investigating' where id=%s;" %(data['BugID'], data['id'])
+        ids = data['id'][:-1]
+        sql = "update alerts set bug='%s',status='Investigating' where id IN(%s);" %(data['BugID'], ids)
```

Comment:
I don't think we need any quotes around %s, its working fine without them .
I assume by data['id'] == '' you mean if no alerts are selected, in that case post request will not be made https://github.com/jmaher/alert_manager/blob/master/js/alerts.js#L330